Sources/Common/DataModel/ImageData/index.js

```diff
@@ -104,7 +104,7 @@ function vtkImageData(publicAPI, model) {
 
   publicAPI.getPoint = (index) => {
     const dims = publicAPI.getDimensions();
-    const ijk = vec3.fromValues(0, 0, 0);
+    const ijk = new Float64Array(3);
```

Comment:
You can do pre-initialization like that `Float64Array.from([0,0,0])`

Reply:
Yes, but native arrays default to 0 so I remeoved it